models/migrations/v99.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+func specifyWebhookSignatureType(x *xorm.Engine) error {
+	var err error
+
+	switch x.Dialect().DriverName() {
+	case "mysql":
```

Comment:
Two things. 1. Rename columns is tricky (for example SQLite also needs to be added to this migration) so perhaps instead of a rename we keep signature column with same content we just have the strict field be renamed, and then add the new sha column. 2. Could you add the adding of the new column to this migration, while it is added by the default migration,sometimes we’ve been bitten by changing a column that doesn’t yet exist so usually we request explicit add of new columns.

Apologies if I am unclear, I am on mobile and can give better details when I get to a computer. Please ask any questions.